src/main/java/com/suse/matcher/facts/Subscription.java

```diff
@@ -107,6 +107,9 @@ public String toString() {
     /** Does this subscription on its own represent a hard bundle? */
     public Boolean singleSubscriptionHardBundle = false;
 
+    /** Id of subscription to which this subscription has been aggregated **/
+    public Long aggregationSubscriptionId;
```

Comment:
(perhaps I am missing something) - shouldn't this be called `aggregatedSubscriptionId`?

Reply:
Hmm, I think `aggregated...` sounds a bit better. lemme change it.